### ISSUE TITLE
Elements are not updating inside Parallax 

### ISSUE THREAD

**Issue by pmadar** [RRutsche/react-parallax#9]
Everything in Parallax compoment are not updating at changing state or receiving new props.
For example:

``` javascript
<Parallax bgImage="goals.jpg" strength={400}>
    <div className="goals">
      {this.props.goals}
   </div>
</Parallax>
```

On receiving new goals nothing happens.

**Comment by rrutsche** [comment]
@pmadar Thanks for the report. Fixed the issue in version [0.4.1](https://github.com/RRutsche/react-parallax/releases/tag/v0.4.1)

**Comment by pmadar** [comment]
Tested. Working great, thank for the super fast fix :)
